executor/insert_common.go

```diff
@@ -1146,6 +1150,7 @@ func (e *InsertValues) batchCheckAndInsert(ctx context.Context, rows [][]types.D
 	}
 
 	if e.stats != nil {
+		e.stats.FKCheckTime += prefetchStart.Sub(start)
```

Comment:
FKCheckTime includes the txn active time, may be not so accurate to indicate "fk check time"

Reply:
The txn is already been activated when the executor open, so it won't cost many time here.